Stop creating symlink to com.apple.webkit.webpushd.plist

msaboff opened the pull request:
#### c3c8a7d837d9b7f9b11b234a20f87931ccdf5c7c
<pre>
Stop creating symlink to com.apple.webkit.webpushd.plist
<a href="https://bugs.webkit.org/show_bug.cgi?id=243213">https://bugs.webkit.org/show_bug.cgi?id=243213</a>
&lt;rdar://89331799&gt;

Reviewed by Saam Barati.

Launchd has been updated to look for daemon plist files in the app cryptex so we no longer
need to create a symlink in /System/Library/LaunchAgents.

* Source/WebKit/Configurations/Base.xcconfig:
* Source/WebKit/Scripts/copy-launchd-plist-and-create-symlink.sh:
* Source/WebKit/Scripts/copy-launchd-plist.sh: Renamed from Source/WebKit/Scripts/copy-launchd-plist-and-create-symlink.sh.
* Source/WebKit/WebKit.xcodeproj/project.pbxproj:

Canonical link: <a href="https://commits.webkit.org/252915@main">https://commits.webkit.org/252915@main</a>
</pre>

emw-apple reviewed (commented):
r=me

Can we rename `copy-launchd-plist-and-create-symlink.sh`, now that it does not actually create a symlink?

saambarati reviewed (approved):
r=me too with Elliott's suggestion

msaboff committed:
Stop creating symlink to com.apple.webkit.webpushd.plist
https://bugs.webkit.org/show_bug.cgi?id=243213
<rdar://89331799>

Reviewed by Saam Barati.

Launchd has been updated to look for daemon plist files in the app cryptex so we no longer
need to create a symlink in /System/Library/LaunchAgents.

* Source/WebKit/Configurations/Base.xcconfig:
* Source/WebKit/Scripts/copy-launchd-plist-and-create-symlink.sh:
* Source/WebKit/Scripts/copy-launchd-plist.sh: Renamed from Source/WebKit/Scripts/copy-launchd-plist-and-create-symlink.sh.
* Source/WebKit/WebKit.xcodeproj/project.pbxproj:

Canonical link: https://commits.webkit.org/252915@main
